src/Voron/Data/Tables/TableSchema.IndexDefinitions.cs

```diff
@@ -221,6 +226,11 @@ public class DynamicKeyIndexDef : AbstractTreeIndexDef
                 }
             }
 
+            public void OnIndexEntryChanged(Transaction tx, Slice key, long oldSize, long newSize)
```

Comment:
maybe we should incline it aggressively?

Reply:
done